modules/javafx.web/src/main/legal/icu_web.md

```diff
@@ -508,3 +443,40 @@ publicity pertaining to distribution of the software without specific,
 written prior permission.  M.I.T. makes no representations about the
 suitability of this software for any purpose.  It is provided "as is"
 without express or implied warranty.
+----------------------------------------------------------------------
```

Comment:
Add blank line above and below this line.

Reply:
Done